ivy/functional/frontends/numpy/mathematical_functions/extrema_finding.py

```diff
@@ -260,3 +260,25 @@ def _fmax(
     if ivy.is_array(where):
         ret = ivy.where(where, ret, ivy.default(out, ivy.zeros_like(ret)), out=out)
     return ret
+
+@handle_numpy_out
+@to_ivy_arrays_and_back
+@from_zero_dim_arrays_to_scalar
```

Comment:
@handle_numpy_dtype
 @handle_numpy_casting

should both exists, please referenced your degrees function implementation as they have similar parameters